library/src/main/scala/giter8/GitRepository.scala

```diff
@@ -47,7 +47,7 @@ object GitRepository {
     val NativeUrl: Regex = "^(git[@|://].*)$".r
     val HttpsUrl: Regex  = "^(https://.*)$".r
     val HttpUrl: Regex   = "^(http://.*)$".r
-    val SshUrl: Regex    = "^(ssh://.*)$".r
+    val SshUrl: Regex    = "^ssh://(.*)$".r
```

Comment:
Yes the ssh protocol wasn't supported, but the problem is that jgit don't know how to handle a url if it comes with the ssh protocol prefix.

This change just change the regex capture group to remove the prefix part, because is what jgit expects.

If you try to do for instance: `sbt new ssh://git@github.com:lagom/lagom-java.g8` or `g8 ssh://git@github.com:lagom/lagom-java.g8` you will get an error. But it will work if doing `sbt new git@github.com:lagom/lagom-java.g8`.